examples/test_data/download_utils.py

```diff
@@ -0,0 +1,71 @@
+# ----------------------------------------------------------------------------
```

Comment:
Can we combine `examples/python/reconstruction_system/scripts/download_dataset.py` with this file? That script supports:
- single place to specify all download URLs. Clients can access data by name instead of URL.
- Allowing users to list available datasets.
- extracting from archives.

Reply:
Yes, that is the plan. Will do that in subsequent PRs since there are several locations to migrate. I've added a list of files to migrate to this new mechanism in the PR description above.